SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Supported Distributions
+
+The list of ROS 2 distributions that receive security updates is at https://docs.ros.org/en/rolling/Releases.html
+
+## Reporting a Vulnerability
+
+Reports of security vulnerabilities should follow the process outlined at https://ros.org/reps/rep-2006.html
```

Comment:
While the full REP is the _correct_ thing to do, I think you should at least put the email address here.

Reply:
I deliberately did not do that, because much of the information that is recommended by the "template" SECURITY.md is already covered in REP-2006.  Thus I thought it would just be best if we redirected people there.

That said, I don't feel that strongly about it, so if someone else thinks we should put the email address here, I can do that.

Reply:
All right, based on additional feedback I added in most of the text of REP-2006, which includes the email address.